catanatron/game.py

```diff
@@ -179,6 +180,10 @@ def playable_actions(self, player):
         for action in city_possible_actions(player, self.board):
             actions.append(action)
 
+        # Can only do if the player has not already played a development card
```

Comment:
For this, we can probably keep a boolean like you did with the robber -- i.e. `HasPlayedDevCardInCurrentTurn` or something and just gate all the dev card action generation under this.